app/controllers/administration/sum_levels_controller.rb

```diff
@@ -28,8 +28,10 @@ def edit
   end
 
   def update
-    if @sum_level.update_by sum_level_params, current_user
+    if params[:commit] && @sum_level.update_by(sum_level_params, current_user)
       redirect_to sum_levels_path, notice: 'Taso päivitettiin onnistuneesti'
+    elsif @sum_level.update_by(sum_level_params, current_user)
+      render :edit
```

Comment:
Isn't this `elsif` redundant? Why is this here? 

Could you add a comment or remove it.